publish-pre-release-cli.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+const _ = require('lodash');
+const fs = require('fs');
+const branchHelper = require('./src/helpers/branch-helper');
+
+const { runShellVerbose } = require('./run.js');
+
+const series = async (...commands) => {
+    for (const command of commands) {
+        await runShellVerbose(...command);
+    }
+};
+
+let newVersion = '';
+let tagName = '';
+
+const generateTagName = async () => {
```

Comment:
This is not just generating the tag name, it's also updating the package.json file. I'd separate the two to keep the functions more focused and purer.